docs/apache-airflow/executor/debug.rst

```diff
@@ -15,11 +15,59 @@
     specific language governing permissions and limitations
     under the License.
 
+Testing DAGs with dag.test()
+=============================
+
+To debug DAGs in an IDE, you can set up the ``dag.test`` command in your dag file and run through your DAG in a single
+serialized python process.
+
+This approach can be used with any supported database (including a local SQLite database) and will
+``fail fast`` as all tasks run in a single process.
```

Comment:
```suggestion
*fail fast* as all tasks run in a single process.
```
I personally only use `` to highlight code, but not sure if we have a standard here (in Airflow).